kategory-docs/docs/docs/datatypes/ior/README.md

```diff
@@ -6,4 +6,117 @@ permalink: /docs/datatypes/ior/
 
 ## Ior 
 
-TODO
\ No newline at end of file
+`Ior` represents an inclusive-or relationship between two data types.
+This makes it very similar to the [`Either`](/docs/datatypes/either) data type, which represents an "exclusive-or" relationship.
+What this means, is that an `Ior<A, B>` (also written as `A Ior B`) can contain either an `A`, a `B`, or both an `A` and `B`.
+Another similarity to `Either` is that `Ior` is right-biased,
+which means that the `map` and `flatMap` functions will work on the right side of the `Ior`, in our case the `B` value.
+You can see this in the function signature of `map`:
+
+```kotlin
+fun <D> map(f: (B) -> D): Ior<A, D>
+```
+
+We can create `Ior` values using `Ior.left`, `Ior.right` and `Ior.both`:
+
+```kotlin:ank
+import kategory.*
+
+val right = Ior.Right(3)
+
+val left = Ior.Left("Error")
+
+val both = Ior.Both("Warning", 3)
+```
+
+Kategory also offers extension functions for `Ior`, the `leftIor`, `rightIor` and `bothIor`:
+
+```kotlin:ank
+import kategory.*
+
+val right = 3.rightIor()
+
+val left = "Error".leftIor()
+
+val both = ("Warning" to 3).bothIor()
+```
+
+
+When we look at the `Monad` or `Applicative` instances of `Ior`, we can see that they actually requires a `Semigroup` instance on the left side.
+This is because `Ior` will actually accumulate failures on the left side, very similar to how the [`Validated`](/docs/datatypes/validated) data type does.
```

Comment:
accumulate -> accumulates

Reply:
it's: Ior will accumulate, so should be as it is, no?

Reply:
Oh shit, right, my brain skipped the will.